### DISCUSSION TITLE
Inertiajs + react

### DISCUSSION THREAD

**Discussion by yashiroiori** [lunarphp/lunar#1162] [Q&A]
Looks great, any plan to make version with inertiajs + react??
I like so much collaborate with thath version

I have an ERP with inertiajs + react with payments, invoice, inventory, etc, and don't have ecommerce in this moment, I think use this package and test 🤔

**Comment by glennjacobs** [comment]
No plans for React, but we do have plans for a Vue offering which we're working on.
